.github/workflows/setup-release-branch-protection.yml

```
@@ -0,0 +1,140 @@
+name: Setup Release Branch Protection
+
+on:
+  create:
+    branches:
+      - 'release-*'
+
+permissions:
+  contents: read
+  administration: write
```

actionlint (1.7.12)

[error] unknown permission scope "administration". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)